backend/.prettierrc.json

```diff
@@ -0,0 +1,6 @@
+{
+    "singleQuote": true,
+    "trailingComma": "all",
+    "jsxBracketSameLine": false,
+    "bracketSpacing": true
```

Comment:
Normalement prettier et eslint n'ont pas exactement les même ype de règle. Je t'avouerais que j'ai pas creusé (j'ai pris ce que l'on a de base sur nos projet en ts. Tu veux que je supprime ?

Reply:
I'd like to keep those rules. Prettier is useful for auto formatting, there exists a plugin "eslint-prettier" to remove all eslint rules that conflict with prettier rules so that all auto formatted code with prettier will pass an eslint check, I assume that's what we're doing here

Reply:
I'll keep the file. Feel free to add to it/ modify it if you want in the future

PS: the plugin is indeed used in eslint config